client/src/components/FaceDetection.jsx

```diff
@@ -53,18 +54,29 @@ function FaceDetection(props) {
       const drawBox = new faceapi.draw.DrawBox(box);
       drawBox.draw(canvas);
       setLoadingDetection(false);
+      setDetectedFaces(true)
     })
   }
 
+  const handleToggleSwitch = () => {
+    return visibleFaces ? setVisibleFaces(false) : setVisibleFaces(true);
+  }
+
   return (
     <Fragment>
       <div className="image-container">
         { imageURL && 
-          <img src={ imageURL } alt="img" id="imageUpload"/>
+          <Fragment>
+            <img src={ imageURL } alt="img"/>
+            <canvas id="detected-faces" className={ visibleFaces ? "" : "invisible" }/>
+          </Fragment>
         }
       </div>  
-      { imageURL &&
-          <button onClick={ start }>Detect Faces</button>
+      { detectedFaces &&
```

Comment:
Good job with this! I like that the switch isn't rendered until the faces are detected.